staramr/blast/results/BlastHitPartitions.py

```diff
@@ -21,14 +21,10 @@ def append(self, hit):
         :param hit: The hit to add.
         :return: None
         """
-        if hit.get_contig_start() > hit.get_contig_end():
+        if hit.get_genome_contig_start() > hit.get_genome_contig_end():
             raise Exception(
-                "Unsupported condition: contig start > contig end for hit (contig=" + hit.get_contig() + ", start=" +
-                str(hit.get_contig_start()) + ", end=" + str(hit.get_contig_end()) + ")")
-        if hit.get_query_frame() != 1:
-            raise Exception("Unsupported condition: query frame is not 1 for hit (contig=" + hit.get_contig() +
-                            ", start=" + str(hit.get_contig_start()) + ", end=" + str(
-                hit.get_contig_end()) + ", query_frame=" + str(hit.get_query_frame()) + ")")
+                "Unsupported condition: contig start > contig end for hit (contig=" + hit.get_genome_contig_id() + ", start=" +
+                str(hit.get_genome_contig_start()) + ", end=" + str(hit.get_genome_contig_end()) + ")")
 
         partition = self._find_parition(hit)
```

Comment:
I renamed to `_get_existing_partition` in 68104b7795e850aa4cf69a408f3adc10f04116f2, since that is what I am trying to do (and if it doesn't exist, create it).

Reply:
Oh I just meant there was a typo haha

Reply:
Oh, thanks.  I didn't even notice.  At least it's fixed now.